### PR TITLE
Translation to Spanish: Update Gear Up for Success

### LINKED ISSUE #6923: Updating Spanish translation on new challenges
- [ ] Basic Front End Development Projects
- [ ] new Gear Up For Success challengesWe're introducing some new challenges

### PR DESCRIPTION
**Pull request by cuent**
Translate to Spanish new challenges of Update Gear Up for Success  file. 
Closes #6923.

### CODE REVIEW

**Comment by BerkeleyFalse** [comment]
@cuent thanks for the PR.
Your branch name should start with one of `fix/, feature/` prefixes. Name, your branches correctly next time, please.
Do not include issue numbers and following [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) in commit messages.
Please, review our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 1]
```diff
@@ -44,13 +44,13 @@
...
-      "nameEs": "Translate",
+      "nameEs": "Únete a nuestro Subreddit",
       "descriptionEs": [
         [
           "http://i.imgur.com/DYjJuCG.gif",
-          "",
-          "",
-          ""
+          "Un gif que muestra como crear una cuenta de Reddit y unirse a Free Code Camp subreddit",
+          "Nuestra comunidad tiene su propio subreddit en Reddit. Es una manera conveniente de hacer preguntas y compartir enlaces con nuestra entera comunidad. Si aún no dispones de una cuenta de Reddit, puedes crear una en pocos segundos - ni siquiera necesitas una dirección de correo electrónico. Entonces puedes hacer clic en el botón de \"subscribe\" para unirte a nuestro subreddit. También puede que te interesé suscribirte a otros subreddits listados en la columna lateral.",
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: entera comunidad -> comunidad entera

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 2]
```diff
@@ -76,19 +76,19 @@
...
-      "nameEs": "Translate",
+      "nameEs": "Lee noticias de Codificación en nuestro canal de publicaciones Medium",
       "descriptionEs": [
         [
           "http://i.imgur.com/FxSOL4a.gif",
-          "",
-          "",
-          ""
+          "Un gif mostrando como crear una cuenta en Medium.",
+          "Nuestra comunidad tiene un canal para publicaciones Medium, donde escribimos un montón de artículos sobre desarrollo de software. Si aún no tienes una cuenta Medium, puedes hacer clic en el enlace y registrarte con tu red social o ingresando tu dirección de correo electrónico (ellos enviarán un correo electrónico que debes hacer clic para crear tu cuenta.) Luego selecciona tus temas de interés y puedes continuar con los siguientes pasos.",
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: hacer clic -> pulsar (no es obligatoria, pero es la sugerencia del glosario).

File: seed/challenges/01-front-end-development-certification/gear-up-for-success.json [hunk 3]
```diff
@@ -76,19 +76,19 @@
...
+          "https://www.medium.com"
         ],
         [
           "http://i.imgur.com/zhhywSX.gif",
-          "",
-          "",
-          ""
+          "Un gif mostrando como hacer clic en el botón \"follow\" para seguir las publicaciones de Free Code Camp.",
+          "Una vez que inicies sesión puedes ir al canal de publicaciones Medium de Free Code Camp y hacer clic en \"follow\". Nuestros campistas publican varios artículos cada semana.",
```
Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/gear-up-for-success.json]: hacer clic -> pulsar

**Pull request closed without merging**